IPython/core/display.py

```diff
@@ -1338,3 +1448,15 @@ def set_matplotlib_close(close=True):
     from ipykernel.pylab.config import InlineBackend
     cfg = InlineBackend.instance()
     cfg.close_figures = close
+
+
+
+for oname in __all__:
+    # set the module name to IPython.display or sphinx cannot correctly cross
+    # link. Mostly it figures out that objets are in IPython.core.display, but
+    # that's a module we explicitly do not document.
+    try:
+        # using hasattr to check still raises an AttributeError.
+        locals()[oname].__module__ = 'IPython.display' 
```

Comment:
Or, we re-allow `IPython.core.display` to be documented. Right now sphinx can't link because there is no `IPython.core.display` page.